### DISCUSSION TITLE
Reduce mortar precision

### DISCUSSION THREAD

**Discussion by Souza-R** [Vdauphin/HeartsAndMinds#1181] [Question and answer]
Hi! I created a mission against the taliban and they have a fucking accuracy using mortars.. it’s impossible to escape .. is there a way to reduce the precision? Sorry about my bad English

**Comment by Vdauphin** [comment]
Hello,
I have no idea
H&M only change skills defined in mission parameter
Cheers

**Reply by Souza-R** [reply]
Can I blacklist mortas ?

**Reply by Vdauphin** [reply]
There are already blacklisted
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L646-L647
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/mil/class.sqf#L109-L117

**Reply by Souza-R** [reply]
Hey! Thanks.. u are always helping me!! 

I don’t think it is static.. sometimes one enemy of the patrol have a mortar on the back ( like a backpack ) and assemble whenever they spot us

**Reply by Vdauphin** [reply]
Then you can filter the class name of the unit here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L649-L670
